src/Apps/Legion/Adapters/Standalone/DataManagement/Models/DataManagementAdapter.types.ts

```diff
@@ -0,0 +1,37 @@
+import { AdapterReturnType } from '../../../../../../Models/Constants/Types';
+import { IBaseAdapter } from '../../../../Models/Interfaces';
+import { DataManagementAdapterData } from './DataManagementAdapter.data';
+
+export interface IDataManagementAdapter extends IBaseAdapter {
+    /** ambigious connection source; can be either a string or an object type or totally different structure, e.g. Kusto cluster url */
+    connectionSource: any;
+    /** ambigious get data method; can have very custom implementation through params passed, e.g. getting Kusto tables using database name */
+    getData: (
```

Comment:
yeah so i wouldnt make these ambiguous.  lets just make the function name descriptive instead of relying on forking based on the parameters passed in

Reply:
yeah, my intention was like creating this IDataManagementAdapter interface to encapsulate all possible data management services and keep it as generic as possible for the future. But okay, I am going to break it into methods as commented below in this file.

Reply:
I agree with Matt, I think it is ok to have a method for getTwinData or something descriptive and each standalone adapter implements that method differently.

Reply:
yeah i think thats what we want.  irem was tryna keep it generic in case there are implementers of adapters that dont need createTable for example, but since those return void anyway we adapters that dont need that can just have a no-op implementation

Reply:
but of course for those which returns a specific adapter data (non void), it might be a problem. But we can make changes in the interface as necessary.